README.md

```diff
@@ -49,79 +74,93 @@ The plugin activates on `.res` and `.resi` files. If you've already got Reason-L
+| ReScript: Open the compiled JS file for this implementation file | Opens the compiled JS file for the current ReScript file.                                                                                                                                                                                                                                             |
+| ReScript: Switch implementation/interface                        | Switches between the implementation and interface file. If you're in a `.res` file, the command will open the corresponding `.resi` file (if it exists), and if you're in a `.resi` file the command will open the corresponding `.res` file. This can also be triggered with the keybinding `Alt+O`. |
+| ReScript: Start Code Analyzer                                    | This will start code analysis in the ReScript project of the file you run the command from.                                                                                                                                                                                                           |
 
-#### `> ReScript: Create an interface file for this implementation file.`
+## 🔨 Settings
 
-Creates an interface file (`.resi`) for the current `.res` file, automatically filling in all types and values in the current file.
+You'll find all ReScript specific settings under the scope `rescript.settings`.
 
-#### `> Open the compiled JS file for this implementation file.`
+| Setting                        | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| ------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Autostarting ReScript builds   | If there's no ReScript build running already in the opened project, the extension will prompt you and ask if you want to start a build automatically. You can turn off this automatic prompt via the setting `rescript.settings.askToStartBuild`.                                                                                                                                                                                                                                    |
```

Comment:
I don't think this title is accurate, since the setting is just about the prompt, not actually autostarting a build. Maybe change it to "Prompt to start build" or something like that?

Reply:
Done https://github.com/rescript-lang/rescript-vscode/pull/508/commits/7086e831685760c7b8cadb4dddf454f4c4d0e2de